lib/learn_test/strategies/python_unittest.rb

```diff
@@ -0,0 +1,67 @@
+require_relative 'python_unittest/requirements_checker'
+require_relative 'python_unittest/nose_installer'
+
+module LearnTest
+  module Strategies
+    class PythonUnittest < LearnTest::Strategy
+      def service_endpoint
+        '/e/flatiron_unittest/build/ironboard'
+      end
+
+      def detect
+        runner.files.any? {|f| f.match(/.*.py$/) }
+      end
+
+      def check_dependencies
+        LearnTest::PythonUnittest::RequirementsChecker.check_installation
```

Comment:
ah yeah, i'll clean that up, i was just bulk moving a bunch of stuff, so i just kept it named the same